### DISCUSSION TITLE
Already created PR is not recognised and not updated

### DISCUSSION THREAD

**Discussion by ganchkal** [renovatebot/renovate#28410] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub, version 37.295.0

### Please tell us more about your question or problem

Hello! Apologies if this is already covered or discussed, but I have a hard time configuring Renovate bot. I am running self-hosted Renovate bot in a separate for the bot repository in GitHub and scan remote repository for npm dependencies updates. On initial run Renovate is creating a PR with the proper versions in description and proper updates in package.json and yarn.lock. On next runs(there is another repo with libraries our generation and the bot is triggered when a new version is pushed), Renovate tries to create again the same PR and its failing with **Response code 422 (Unprocessable Entity)** **A pull request already exists for...** . However, the files in the PR are updated properly, package.json and yarn.lock. Is this a proper behaviour? I expect when a PR is already created, the bot to find it and modify it properly. Is there a way to modify the description as well with up-to-date versions and/or adding comments? I am attaching the configs as Logs.

### Logs (if relevant)

<details><summary>Logs</summary>

```

DEBUG: File config
       "config": {
         "platform": "github",
         "branchPrefix": "renovate/",
         "commitMessagePrefix": "chore(deps): WP-0000 ",
         "username": "renovate-bot",
         "gitAuthor": "Renovate Bot <renovatebot@email.com>",
         "repositories": ["my-repo"],
         "extends": ["config:recommended"],
         "enabledManagers": ["npm"],
         "useBaseBranchConfig": "merge",
         "prHourlyLimit": 5
       }
DEBUG: CLI config
       "config": {}
DEBUG: Env config
       "config": {"hostRules": [], "token": "***********"}
DEBUG: Combined config
       "config": {
         "platform": "github",
         "branchPrefix": "renovate/",
         "commitMessagePrefix": "chore(deps): WP-0000 ",
         "username": "renovate-bot",
         "gitAuthor": "Renovate Bot <renovatebot@email.com>",
         "repositories": ["my-repo"],
         "extends": ["config:recommended"],
         "enabledManagers": ["npm"],
         "useBaseBranchConfig": "merge",
         "prHourlyLimit": 5,
         "hostRules": [],
         "token": "***********"
       }

-----------

DEBUG: Repository config (repository=my-repo)
       "fileName": "renovate.json",
       "config": {
         "$schema": "https://docs.renovatebot.com/renovate-schema.json",
         "commitMessagePrefix": "chore(deps): WP-0000 ",
         "packageRules": [
           {"matchPackagePatterns": ["*"], "enabled": false},
           {
             "matchPackagePatterns": [
               "^@sentry/",
               "^lodash",
               "^@types/lodash",
               "^@snowplow/"
             ],
             "enabled": false
           },
           {"matchDepTypes": ["peerDependencies"], "enabled": false},
           {
             "matchPackagePatterns": ["^@ps/web-"],
             "groupName": "web packages",
             "automerge": true,
             "enabled": true
           },
           {
             "matchPackagePatterns": ["^@ps/web-"],
             "matchUpdateTypes": ["major"],
             "enabled": false
           }
         ]
       }
```

</details>

**Comment by rarkins** [comment, marked as answer]
Your `username` is wrong. You're probably best to leave it out. Also leave out `gitAuthor` if you're new and not sure what you're doing. Renovate doesn't know how to find its own PRs because you're telling it the wrong `username` to look for

**Reply by ganchkal** [reply]
You are right. As I am using GitHub App with generated PAT every time, the username and gitAuthor were out of sense. Works like a charm now. Thank you a lot!
